doc/releases/changelog-dev.md

```diff
@@ -1,623 +1,19 @@
 :orphan:
```

Comment:
This file is now an empty changelog, its original content was moved to `doc/releases/changelog-0.20.0.md`. The only change made in that file is marking the release there as the "current release". Further changes will be upcoming when merging #1977 into the release candidate branch and merging the release candidate branch into `master`.